packages/iov-bns/README.md

```diff
@@ -16,14 +16,14 @@ Simplest usage, to use auto-detecting tendermint client and standard bns
 transaction parser:
 
 ```ts
-const connection = await BnsConnection.establish("wss://bns.hugnet.iov.one");
+const connection = await BnsConnection.establish("wss://rpc.lovenet.iov.one");
```

Comment:
When should `bns` be used and when `rpc`?

Reply:
`bns.` was renamed to `rpc.` to express the fact that we are not running two blockchains per network anymore. The first networks that use the new name are
* Public testnet: 😍net (https://github.com/iov-one/ops/issues/187)
* Internal testnet: 🐗net (https://github.com/iov-one/ops/issues/186)